test/language/statements/let/let-newline-await-in-normal-function.js

```diff
@@ -0,0 +1,20 @@
+// |reftest| error:SyntaxError
```

Comment:
this is for spidermonkey, right? I believe it will and should parse the negative flag already in the frontmatter